### ISSUE TITLE
white page when build release

### ISSUE THREAD

**Issue by taduyde** [microsoft/react-native-windows#10300]
### Problem Description

![image](https://user-images.githubusercontent.com/8236997/180600898-42eecfaf-2cd7-46ed-8a5f-a14fae0b7d13.png)

### Steps To Reproduce

run command: npx react-native run-windows --logging --release

### CLI version

npx react-native --version

### Environment

```markdown
npx react-native info
```

**Comment by taduyde** [comment]
any update for this?

**Comment by nguyenluminhvy** [comment]
I have same problem !

**Comment by chrisglein** [comment]
@jonthysell Which issue are we using to track this?

**Comment by chrisglein** [comment]
Duplicate of #10255

**Comment by jonthysell** [comment]
Yes, this is a duplicate of #10255 and will be resolved by #10298.
